fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/GroupByTest.java

```diff
@@ -289,7 +293,7 @@ void testBitMapWithBitMapIndex() {
                 Optional.empty(),
                 IndexQueryabilityFilter.TRUE,
                 EvaluationContext.empty()).getPlan();
-        assertBitMapResult(hook, plan, 1250);
+        assertBitMapResult(hook, plan, 1);
```

Comment:
I think this should be `IndexOptions.BITMAP_VALUE_ENTRY_SIZE_OPTION / 8`?

Reply:
The returned bitmap size (in bits) is determined in the constructor of `BitmapValueIndexMaintainer`:

https://github.com/FoundationDB/fdb-record-layer/blob/09ab9fde82fa51f67c188a303a0b370237bcfd3b/fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/BitmapValueIndexMaintainer.java#L100-L101

So, according to the test, it should be `4`. Now, the actual byte array size is determined as the following:

https://github.com/FoundationDB/fdb-record-layer/blob/09ab9fde82fa51f67c188a303a0b370237bcfd3b/fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/BitmapValueIndexMaintainer.java#L203

So, we should get 1 byte back in return, which is exactly correct.